src/table.js

```diff
@@ -578,13 +578,53 @@ Table.prototype.delete = function(gaxOptions, callback) {
   );
 };
 
+/**
+ * Truncate the table.
+ *
+ * @param {object} [gaxOptions] Request configuration options, outlined
+ *     here: https://googleapis.github.io/gax-nodejs/CallSettings.html.
+ * @param {function} callback The callback function.
+ * @param {?error} callback.err An error returned while making this request.
+ * @param {object} callback.apiResponse The full API response.
+ *
+ * @example
+ * table.truncate(function(err, apiResponse) {});
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * table.truncate().then(function(data) {
+ *   var apiResponse = data[0];
+ * });
+ */
+Table.prototype.truncate = function(gaxOptions, callback) {
+  if (is.fn(gaxOptions)) {
+    callback = gaxOptions;
+    gaxOptions = {};
+  }
+
+  var reqOpts = {
+    name: this.id,
+    deleteAllDataFromTable: true,
+  };
+
+  this.bigtable.request(
+    {
+      client: 'BigtableTableAdminClient',
+      method: 'dropRowRange',
+      reqOpts: reqOpts,
+      gaxOpts: gaxOptions,
+    },
+    callback
+  );
+};
+
 /**
  * Delete all rows in the table, optionally corresponding to a particular
  * prefix.
  *
  * @param {object} [options] Configuration object.
- * @param {string} [options.prefix] Row key prefix, when omitted all rows
- *     will be deleted.
+ * @param {string} [options.prefix] Row key prefix.
```

Comment:
Can you add a comment that this is required?

Reply:
That's a good point-- Can the signature of the method just be a prefix and callback?

```js
table.deleteRows('prefix', callback)
```

Reply:
Don't we need to be able to pass in gaxOptions?

Reply:
Another good point! Looks like I forgot to document that, so it slipped my mind.

However, in cases like this, we would have `gaxOptions` be a standalone argument, so that we use the most predictable path possible:

```js
// 99 percent of the time
table.deleteRows({ prefix: 'prefix' }, callback)

// 1% of the time
table.deleteRows({ prefix: 'prefix', gaxOptions: {...} }, callback)

// So why not
table.deleteRows('prefix', callback)
```

Reply:
Done